lib/pharos/host/configurer.rb

```diff
@@ -96,6 +96,35 @@ def cluster_config
         self.class.cluster_config
       end
 
+      # @return [Pharos::SSH::File]
+      def env_file
+        @ssh.file('/etc/environment')
+      end
+
+      def update_env_file
+        return if @host.environment.nil? || @host.environment.empty?
```

Comment:
Is this one-way thing? One can add envs but never remove them?

Reply:
Yes. If the `original_data.merge(@host.environment).compact.map` below would be changed to something that would not overwrite `nil`'s during merge, you could delete by setting `environment: FOOFOO: null`.

Reply:
Now changed to do so.

```ruby
{ a: nil, b: 'foo' }.merge(a: 'foo', c: 'bar') { |a,b| b }
=> {:a=>nil, :b=>"foo", :c=>"bar"}
{ a: nil, b: 'foo' }.merge(a: 'foo', c: 'bar') { |a,b| b }.compact
=> {:b=>"foo", :c=>"bar"}
```

Reply:
Specs would be nice.

Reply:
Whoomp, there it is.